### PR TITLE
Added mkvtoolnix

### PR DESCRIPTION
**Pull request by lcmen**

### CODE REVIEW

File: Casks/mkvtoolnix.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'mkvtoolnix' do
+  version '7.4'
+  sha256 '886e54887e7bba966c2838afb6b6eb98fc9e4ce3853cce9adb05774b826433ef'
+
+  url 'https://www.macupdate.com/download/16837/MKVToolNix-7.4.0.dmg'
+  name 'MKVToolnix'
+  homepage 'https://www.bunkus.org/videotools/mkvtoolnix'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (iggyvolz) [inline Casks/mkvtoolnix.rb]: Please add a license if possible.

**Comment by vitorgalvao** [comment]
Thank you but this cask [has already been removed](https://github.com/caskroom/homebrew-cask/pull/6516).

We’re considering its [addition to caskroom/unofficial](https://github.com/caskroom/homebrew-unofficial/issues/23).

**Comment by lowski** [comment]
@vitorgalvao can I ask why? I'm using MkvToolnix very often and I would like to install it via Cask.

**Comment by vitorgalvao** [comment]
Sure. The issue is [the download url isn’t official](https://www.bunkus.org/videotools/mkvtoolnix/downloads.html#macosx), and [such casks belong in the aforementioned repo](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#unofficial-vendorless-or-walled-builds). If you add it there you can still use homebrew-cask to install it. It’s simply a matter of tapping another repo.

**Comment by iggyvolz** [comment]
>remove mkvtoolnix, no official binaries, only built by individuals

**Comment by iggyvolz** [comment]
If you want to install it from homebrew, you can simply run brew install --with-wxmac mkvtoolnix.

**Comment by lowski** [comment]
@iggyvolz that's not the sime ```brew install --with-wxmac mkvtoolnix``` will install cli tool (+ bunch of other dependencies). The one here is a binary version GUI tool.

@vitorgalvao in that case I will add it to [homebrew-unofficial](https://github.com/caskroom/homebrew-unofficial).

**Pull request closed without merging**